VR Maze/Assets/Scripts/WalkingScript.cs

```diff
@@ -1,55 +1,53 @@
 ﻿using UnityEngine;
 using System;
 
-public class WalkingScript : MonoBehaviour {
-
-	public GameObject cameraView;
-	public Rigidbody playerRigidbody;
-	public CapsuleCollider playerCollider;
-
-	private float currentSpeed = 0;
-	private float tempSpeed = 0;
-	private float camViewRotX;	//The value of the camera when you look up or doww
-
-	void Update(){
-
-		camViewRotX = cameraView.transform.eulerAngles.x; //Angel of the camer >0 is looking down <0 looking up
-
-
-		if (camViewRotX > 8 && camViewRotX < 90) {
-			speedUp (camViewRotX);
+namespace Assests.Scripts
```

Comment:
Really minor, but this is misspelled from "Assets.Scripts", and the misspelling will mean that this class can't access anything in the Assets.Scripts namespace, and likewise, nothing else can access this (without imports). It looks like WalkingScript and Speed are the only things affected. Doesn't break anything now, but could cause issues in the future